components/content_snare/actions/create-request/create-request.mjs

```diff
@@ -0,0 +1,91 @@
+import app from "../../content_snare.app.mjs";
+
+export default {
+  key: "content_snare-create-request",
+  name: "Create Request",
+  description: "Initiates a novel request on Content Snare. The mandatory prop is name and all other props are optional. [See the documentation](https://api.contentsnare.com/partner_api/v1/documentation#post-/partner_api/v1/requests)",
```

Comment:
I don't think it's necessary to include info about the props. Also, the info doesn't appear to be correct. In addition to `name`, the props `clientEmail` and `clientFullName` are also set as required.
```suggestion
  description: "Initiates a novel request on Content Snare. [See the documentation](https://api.contentsnare.com/partner_api/v1/documentation#post-/partner_api/v1/requests)",
```